### DISCUSSION TITLE
[A-L] (12 Giugno 2023) Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#388] [Esami A-L]
Una relazione binaria $\Large R$ su un insieme $\Large A$ è detta simpatica se ha la seguente proprietà:

<div align="center">

per ogni $\Large a,\ b,\ c,\ d \in A$, se $\Large aRb$ e $\Large bRc$ e $\Large cRd$ allora $\Large aRd$.

</div>

Indicare se i seguenti punto sono veri o falsi.

1. Ogni relazione transitiva è simpatica. [1.5 punti]

2. Ogni relazione simpatica è transitiva. [1.5 punti]

3. Nessuna relazione simpatica è transitiva. [2 punti]

**Comment by cek753** [comment]
V- se aRb, bRc -> aRc, cRd->aRd(quindi è simpatica)
F- dato che per essere simpatica basta avere aRb, bRc, cRd, aRd ma non aRc (quindi non è transitiva)
F
